script/deploy/facets/UpdateCoreFacets.s.sol

```diff
@@ -162,6 +144,8 @@ contract DeployScript is UpdateScriptBase {
             buildInitialCut(selectors, calldVerifAddress);
         }
 
+        console.log("facet cuts successfully generated");
+
```

Comment:
Is the console.log really needed here?

Reply:
it helps to narrow down where errors occur. I added it for that purpose.
It's just a script, not a contract we deploy...